gems/pending/appliance_console/service_group.rb

```diff
@@ -49,7 +49,7 @@ def stop
     private
 
     def enable_miqtop
-      LinuxAdmin.run("chkconfig", :params => {"--add" => "miqtop"})  # Is this really needed?
+      LinuxAdmin::Service.new("miqtop").enable
```

Comment:
@carbonin Seems like this does not belong with this PR as it is a different change.  Suggest moving this to a separate PR.